_rules/table-headers-attribute-refer-to-data-cells.md

```diff
@@ -0,0 +1,208 @@
+---
+name: `headers` attribute only refers to `cells` in the same `table` element
+
+test_type: atomic
+
+description: |
+  This rule checks that the `headers` attribute must only refer to `cells` in the same `table` element
+
+test_aspects:
+- DOM Tree
+
+authors:
+- Jey Nandakumar
+- Audrey Maniez
+---
+
+## Test Procedure
+
+### Applicability
+
+This rule applies to `td` and `th` elements with a `headers` attribute and within a `table` element, that is either [visible](#visible) or [included in the accessibility tree](#included-in-the-accessibility-tree).
```

Comment:
I don't think this should test cells that are excluded from the accessibility tree.

Reply:
I agree. Cells that are excluded don't need correct associations as AT don't (or at least should not) communicate their relationships anyway.